### DISCUSSION TITLE
Create a config to allow pack makers to adjust the limits of enchantment combination

### DISCUSSION THREAD

**Discussion by MuteTiefling** [InnovativeOnlineIndustries/Industrial-Foregoing#1149] [Completed/Done]
Create a config to allow pack makers to adjust the limits of enchantment combination.

This would allow pack makers to decide what they feel is balanced in terms of enchanting.

**Comment by TemperedFool** [comment]
Yes please! This would be very helpful for configuring modpacks and balancing various magic-related mods!

**Comment by jebjordan** [comment]
@Buuz135 Is it at all possible to make the default as true?
in other words, keep the same default behaviour as it was before, but just *allow* pack devs to enable the limit?
